### DISCUSSION TITLE
$request variable returning values from another Project

### DISCUSSION THREAD

**Discussion by mabdalla123** [laravel/framework#44655] [Q&A]
Previously i was working on a  Result System  for a university [link](https://github.com/mabdalla123/ResultSystem)
and  i have started another project [Link](https://github.com/mabdalla123/Laravel-daily-Junior-Test-Project) , and when i started with api login point  which contain the following 

` 
public function __invoke(LoginRequest $request)

    {

        //return $request;
        
        if (Auth::attempt($request->only(["email","password"]))) {
           
            $user = User::where('email', $request['email'])->first();
            //create Token
            $token = $user->createToken('Api');

            return response(
                [
                    'user' => $user,
                    'token' => $token->plainTextToken,
                ],
                200,
            );
        }

        return response(
            'User Not Found',
            404,
        );
    }` 

when i just  return the variable   i get the following data 
`{
    "student_id": 1,
    "semester_id": 1,
    "average": 100,
    "data": "yes",
    "details": [
        {
            "subject_id": 1,
            "student_certified_hours": 1,
            "avarege": 1,
            "mark": "A",
            "point": 6
        },
        {
            "subject_id": 2,
            "student_certified_hours": 1,
            "avarege": 1,
            "mark": "A",
            "point": 6
        }
    ],
    "email": "malika.purdy@example.org",
    "password": "'admin123'"
}`

why does the $request variable have this data from another project?
please note that  when i started the new projects i used the same database and i a have dropped the old tables so this data is not related to database

**Comment by henzeb** [comment]
If you use the same cache service and do not have distinctive enough names, this can be your issue. Try changing the app name in your .env

**Reply by mabdalla123** [reply]
i have tried :
-changing the name of the  app 
-php artisan cache:clear
-php artisan config:clear
nothing changed
